plugins/tds61.js

```diff
@@ -644,6 +644,9 @@ tds61 = {
         // Drop the FCSUBTYPE since we don't use it
         if (attrs.FCSUBTYPE) delete attrs.FCSUBTYPE;
 
+        // Drop the the "Not Found" F_CODE. This is from the UI
+        if (attrs.F_CODE == 'Not found') delete attrs.F_CODE;
+
```

Comment:
I confirmed the UI is unaffected by the code that deletes F_CODE = 'Not found', but if it's needed in TDSv61, why isn't it also needed in TDSv40 and MGCP?